python/sauron-core/core/analysis/stats.py

```
@@ -0,0 +1,197 @@
+import numpy as np
+import pandas as pd
+from typing import Dict, List, Tuple
+from scipy.stats import entropy
+from sklearn.preprocessing import StandardScaler
+
+class StatisticalAnalyzer:
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)